introduction.md

```diff
@@ -7,3 +7,4 @@ Deadly skills:
 * Javascript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?